org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/wf/WWFActivity.java

```diff
@@ -656,11 +657,14 @@ else if (MWFNode.ACTION_UserChoice.equals(node.getAction()))
 					m_activity.setUserChoice(AD_User_ID, value, dt, textMsg);
 					MWFProcess wfpr = new MWFProcess(m_activity.getCtx(), m_activity.getAD_WF_Process_ID(), m_activity.get_TrxName());
 					wfpr.checkCloseActivities(m_activity.get_TrxName());
+					
+					if (!Util.isEmpty(m_activity.getProcessMsg(), true))
+						throw new AdempiereException(m_activity.getProcessMsg());

```

Comment:
for consistency, if we add this checking here, we need to add it to the "User Action" block below too.

Reply:
@hengsin done